### ISSUE TITLE
PeterNorth bug

### ISSUE THREAD

**Issue by chempke** [PAhelper/PhoenixAdult.bundle#211]
This one scene gives me critical error
https://www.peternorth.com/en/video/POV-33-Part-1-Scene-01/31380

```
2019-06-07 23:49:20,078 (57dc) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgpkYmUzMGI2NDIxMDc4ZDFlczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczE2ClBldGVybm9ydGggMzEzODBzNApuYW1lczEwMApLJTNBJTVDQ29udGVudCU1Q1BldGVyTm9ydGglNUNQZXRlck5vcnRoJTIwLSUyMFBPViUyMCUyMzMzJTIwUGFydCUyMDElNUNQZXRlck5vcnRoJTIwLSUyMDMxMzgwJTJFbXA0czgKZmlsZW5hbWVzNDAKYzViOTNmNTk0Mjg2YmYxMzA1ZWNiYjNlNjYzMGIxZjk4MzNiMGNkN3M4CnBsZXhIYXNoczcKMTE3MTc0OHM4CmR1cmF0aW9uczYKMjk0NTgxczIKaWRyMAo_
2019-06-07 23:49:20,079 (57dc) :  DEBUG (runtime:49) - Received packed state data (15056 bytes)
2019-06-07 23:49:20,082 (57dc) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgpkYmUzMGI2NDIxMDc4ZDFlczE3Cm9wZW5TdWJ0aXRsZXNIYXNoczE2ClBldGVybm9ydGggMzEzODBzNApuYW1lczEwMApLJTNBJTVDQ29udGVudCU1Q1BldGVyTm9ydGglNUNQZXRlck5vcnRoJTIwLSUyMFBPViUyMCUyMzMzJTIwUGFydCUyMDElNUNQZXRlck5vcnRoJTIwLSUyMDMxMzgwJTJFbXA0czgKZmlsZW5hbWVzNDAKYzViOTNmNTk0Mjg2YmYxMzA1ZWNiYjNlNjYzMGIxZjk4MzNiMGNkN3M4CnBsZXhIYXNoczcKMTE3MTc0OHM4CmR1cmF0aW9uczYKMjk0NTgxczIKaWRyMAo_
2019-06-07 23:49:20,082 (57dc) :  INFO (agentkit:957) - Searching for matches for {'openSubtitlesHash': 'dbe30b6421078d1e', 'name': 'Peternorth 31380', 'filename': 'K%3A%5CContent%5CPeterNorth%5CPeterNorth%20-%20POV%20%2333%20Part%201%5CPeterNorth%20-%2031380%2Emp4', 'plexHash': 'c5b93f594286bf1305ecbb3e6630b1f9833b0cd7', 'duration': '1171748', 'id': '294581'}
2019-06-07 23:49:20,082 (57dc) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294581/tree'
2019-06-07 23:49:20,088 (57dc) :  INFO (logkit:16) - *******MEDIA TITLE****** Peternorth 31380
2019-06-07 23:49:20,088 (57dc) :  INFO (logkit:16) - Getting Search Settings for: Peternorth 31380
2019-06-07 23:49:20,088 (57dc) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation: Peternorth 31380
2019-06-07 23:49:20,092 (57dc) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Peternorth 31380
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - ^^^^^^^ siteID: 384
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - peternorth  vs peter north
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - 2
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - 3
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - 6
2019-06-07 23:49:20,094 (57dc) :  INFO (logkit:16) - searchTitle (before date processing): 31380
2019-06-07 23:49:20,095 (57dc) :  INFO (logkit:16) - Search Title: 31380
2019-06-07 23:49:20,095 (57dc) :  INFO (logkit:16) - 31380
2019-06-07 23:49:20,101 (57dc) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/search/31380'
2019-06-07 23:49:20,918 (57dc) :  DEBUG (networking:161) - Fetching 'http://www.peternorth.com/en/search/31380/2' from the HTTP cache
2019-06-07 23:49:20,944 (57dc) :  DEBUG (networking:161) - Fetching 'http://www.peternorth.com/en/search/31380/1/dvd' from the HTTP cache
2019-06-07 23:49:20,963 (57dc) :  DEBUG (runtime:88) - Sending packed state data (15088 bytes)
2019-06-07 23:49:20,963 (57dc) :  DEBUG (runtime:924) - Response: [200] str, 768 bytes
2019-06-07 23:49:21,551 (5f34) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ1ODFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzODQKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL19lbl92aWRlb19QT1YtMzMtUGFydC0xLVNjZW5lLTAxXzMxMzgwfDM4ND9sYW5nPWVuczQKZ3VpZHM0MgpfZW5fdmlkZW9fUE9WLTMzLVBhcnQtMS1TY2VuZS0wMV8zMTM4MHwzODRzMgppZHIwCg__
2019-06-07 23:49:21,552 (5f34) :  DEBUG (runtime:49) - Received packed state data (15440 bytes)
2019-06-07 23:49:21,553 (5f34) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ1ODFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzODQKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL19lbl92aWRlb19QT1YtMzMtUGFydC0xLVNjZW5lLTAxXzMxMzgwfDM4ND9sYW5nPWVuczQKZ3VpZHM0MgpfZW5fdmlkZW9fUE9WLTMzLVBhcnQtMS1TY2VuZS0wMV8zMTM4MHwzODRzMgppZHIwCg__
2019-06-07 23:49:21,553 (5f34) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://_en_video_POV-33-Part-1-Scene-01_31380|384?lang=en
2019-06-07 23:49:21,555 (5f34) :  DEBUG (model:234) - Deserializing from C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\3\b7a41bd0d7731675146e1c9a07a460dc4c45049.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-07 23:49:21,556 (5f34) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294581/tree'
2019-06-07 23:49:21,573 (5f34) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-07 23:49:21,573 (5f34) :  INFO (logkit:16) - 384
2019-06-07 23:49:21,573 (5f34) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-07 23:49:21,581 (5f34) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/video/POV-33-Part-1-Scene-01/31380'
2019-06-07 23:49:22,494 (5f34) :  INFO (logkit:16) - Directors found: 0
2019-06-07 23:49:22,497 (5f34) :  CRITICAL (core:574) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://_en_video_POV-33-Part-1-Scene-01_31380|384?lang=en' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 1063, in update
    metadata = PAsearchSites.networkGammaEnt.update(metadata,siteID,movieGenres,movieActors)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\networkGammaEnt.py", line 400, in update
    i = int(i) + 1
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2019-06-07 23:49:22,499 (5f34) :  DEBUG (model:229) - Serializing to C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\3\b7a41bd0d7731675146e1c9a07a460dc4c45049.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-07 23:49:22,506 (5f34) :  DEBUG (runtime:88) - Sending packed state data (15472 bytes)
2019-06-07 23:49:22,506 (5f34) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes

```

**Comment by chempke** [comment]
Another one
https://www.peternorth.com/en/video/North-Pole-90-Part-1-Scene-02/33512

```
2019-06-09 22:20:50,421 (df8) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgowODcyYTMyODJmMDFmZjc0czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczE2ClBldGVybm9ydGggMzM1MTJzNApuYW1lczEwOQpLJTNBJTVDQ29udGVudCU1Q1BldGVyTm9ydGglNUNQZXRlck5vcnRoJTIwLSUyME5vcnRoJTIwUG9sZSUyMCUyMzkwJTIwUGFydCUyMDElNUNQZXRlck5vcnRoJTIwLSUyMDMzNTEyJTJFbXA0czgKZmlsZW5hbWVzNDAKNjcxMmU1OTZlMjRhZjZmYzUyOWJmZjk5MTNmZWZjMDc0ODcwNGIyNHM4CnBsZXhIYXNoczcKMTEyMjY4OXM4CmR1cmF0aW9uczYKMjk0OTYxczIKaWRyMAo_
2019-06-09 22:20:50,421 (df8) :  DEBUG (runtime:49) - Received packed state data (12340 bytes)
2019-06-09 22:20:50,424 (df8) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgowODcyYTMyODJmMDFmZjc0czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczE2ClBldGVybm9ydGggMzM1MTJzNApuYW1lczEwOQpLJTNBJTVDQ29udGVudCU1Q1BldGVyTm9ydGglNUNQZXRlck5vcnRoJTIwLSUyME5vcnRoJTIwUG9sZSUyMCUyMzkwJTIwUGFydCUyMDElNUNQZXRlck5vcnRoJTIwLSUyMDMzNTEyJTJFbXA0czgKZmlsZW5hbWVzNDAKNjcxMmU1OTZlMjRhZjZmYzUyOWJmZjk5MTNmZWZjMDc0ODcwNGIyNHM4CnBsZXhIYXNoczcKMTEyMjY4OXM4CmR1cmF0aW9uczYKMjk0OTYxczIKaWRyMAo_
2019-06-09 22:20:50,424 (df8) :  INFO (agentkit:957) - Searching for matches for {'openSubtitlesHash': '0872a3282f01ff74', 'name': 'Peternorth 33512', 'filename': 'K%3A%5CContent%5CPeterNorth%5CPeterNorth%20-%20North%20Pole%20%2390%20Part%201%5CPeterNorth%20-%2033512%2Emp4', 'plexHash': '6712e596e24af6fc529bff9913fefc0748704b24', 'duration': '1122689', 'id': '294961'}
2019-06-09 22:20:50,424 (df8) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294961/tree'
2019-06-09 22:20:50,430 (df8) :  INFO (logkit:16) - *******MEDIA TITLE****** Peternorth 33512
2019-06-09 22:20:50,430 (df8) :  INFO (logkit:16) - Getting Search Settings for: Peternorth 33512
2019-06-09 22:20:50,430 (df8) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation: Peternorth 33512
2019-06-09 22:20:50,434 (df8) :  INFO (logkit:16) - mediaTitle w/ possible abbrieviation fixed: Peternorth 33512
2019-06-09 22:20:50,434 (df8) :  INFO (logkit:16) - ^^^^^^^ siteID: 384
2019-06-09 22:20:50,434 (df8) :  INFO (logkit:16) - ^^^^^^^ Shortening Title
2019-06-09 22:20:50,434 (df8) :  INFO (logkit:16) - peternorth  vs peter north
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - 2
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - 3
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - 6
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - searchTitle (before date processing): 33512
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - Search Title: 33512
2019-06-09 22:20:50,436 (df8) :  INFO (logkit:16) - 33512
2019-06-09 22:20:50,443 (df8) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/search/33512'
2019-06-09 22:20:51,321 (df8) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/search/33512/2'
2019-06-09 22:20:52,165 (df8) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/search/33512/1/dvd'
2019-06-09 22:20:52,937 (df8) :  DEBUG (runtime:88) - Sending packed state data (12372 bytes)
2019-06-09 22:20:52,937 (df8) :  DEBUG (runtime:924) - Response: [200] str, 780 bytes
2019-06-09 22:20:53,750 (6050) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ5NjFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTEKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL19lbl92aWRlb19Ob3J0aC1Qb2xlLTkwLVBhcnQtMS1TY2VuZS0wMl8zMzUxMnwzODQ%40bGFuZz1lbnM0Cmd1aWRzNDkKX2VuX3ZpZGVvX05vcnRoLVBvbGUtOTAtUGFydC0xLVNjZW5lLTAyXzMzNTEyfDM4NHMyCmlkcjAK
2019-06-09 22:20:53,750 (6050) :  DEBUG (runtime:49) - Received packed state data (11340 bytes)
2019-06-09 22:20:53,753 (6050) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyOTQ5NjFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTEKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL19lbl92aWRlb19Ob3J0aC1Qb2xlLTkwLVBhcnQtMS1TY2VuZS0wMl8zMzUxMnwzODQ@bGFuZz1lbnM0Cmd1aWRzNDkKX2VuX3ZpZGVvX05vcnRoLVBvbGUtOTAtUGFydC0xLVNjZW5lLTAyXzMzNTEyfDM4NHMyCmlkcjAK
2019-06-09 22:20:53,753 (6050) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://_en_video_North-Pole-90-Part-1-Scene-02_33512|384?lang=en
2019-06-09 22:20:53,753 (6050) :  DEBUG (model:234) - Deserializing from C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\b\ed4e0737acd31c8c0d3e9a3d64b55cf597bca03.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-06-09 22:20:53,756 (6050) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/294961/tree'
2019-06-09 22:20:53,773 (6050) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-09 22:20:53,773 (6050) :  INFO (logkit:16) - 384
2019-06-09 22:20:53,773 (6050) :  INFO (logkit:16) - ******UPDATE CALLED*******
2019-06-09 22:20:53,779 (6050) :  DEBUG (networking:166) - Requesting 'http://www.peternorth.com/en/video/North-Pole-90-Part-1-Scene-02/33512'
2019-06-09 22:20:54,951 (6050) :  INFO (logkit:16) - Directors found: 0
2019-06-09 22:20:54,956 (6050) :  CRITICAL (core:574) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://_en_video_North-Pole-90-Part-1-Scene-02_33512|384?lang=en' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 1063, in update
    metadata = PAsearchSites.networkGammaEnt.update(metadata,siteID,movieGenres,movieActors)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\networkGammaEnt.py", line 400, in update
    i = int(i) + 1
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-eadbcbb45\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

```

**Comment by dxm2891** [comment]
Fix this bug in commits 321609c 2019-06-25
